src/plugins/dashboard/public/dashboard_container/component/grid/dashboard_grid_item.tsx

```diff
@@ -22,7 +23,6 @@ export interface Props extends DivProps {
   id: DashboardPanelState['explicitInput']['id'];
   index?: number;
   type: DashboardPanelState['type'];
-  focusedPanelId?: string;
```

Comment:
@ThomThomson This prop doesn't actually seem to be used anywhere, so I removed it to reduce confusion.

Reply:
Thanks @cqliu1! I think this was something that was introduced long ago _anticipating_ a focused panel, but the feature didn't exist.